File: src/main/java/com/unicorn/service/FileService.java
package com.unicorn.service;

import com.unicorn.utils.Util;
import org.springframework.stereotype.Service;

import java.io.IOException;

/**
 * Created by dtimchenko on 03-Aug-17.
 */
@Service
public class FileService {

    public String jsonFileContent(String fileName) {
        String jsonContent = "";
        try {
            jsonContent = Util.readFile(fileName);
        } catch (IOException e) {
            e.printStackTrace();
        }
        return jsonContent;
    }
}

File: src/main/java/com/unicorn/restemulator/service/FileService.kt
package com.unicorn.restemulator.service

import com.unicorn.restemulator.util.Utils
import org.springframework.stereotype.Service

/**
 * Created by dtimchenko on 10-Aug-17.
 */

@Service
class FileService {

    fun getJsonFileContent(fileName: String): String {
        return Utils.readFile(fileName)
    }
}